### ISSUE TITLE
Session refactoring

### ISSUE THREAD

**Issue by luke-siedle** [gernzy/gernzy-server#55]
Right now there's a Session model as well as a SessionService. 

The SessionService is confusing because it basically interacts with the one and only Session object that should be available every request. It may be simpler if we use the Session model alone, and also use methods like access the session on the $context object passed into the PHP graphql library resolvers. `$context->request->session` provides this model right now. 

The session gets merged to the request using custom middleware found here:
https://github.com/gernzy/gernzy-server/blob/master/src/Middleware/CartMiddleware.php#L43

You can see this uses the SessionService but could just as easily instantiate the model directly. I feel like my original solution is overengineered and confusing for the time being. 

There are some static helper methods used throughout the code, but these could just be declared on the model itself. 

I don't really foresee a need to swap out the Session object with something else (like a redis store), but if that does need to happen we can always do this later. 

@AlexeiaW let me know your thoughts.

**Comment by AlexeiaW** [comment]
@luke-siedle I agree with you. Sticking to the available session model makes things a bit simpler. Also, using the lighthouse-provided session makes sense. 

I did think about if we ever need to swop the session store, for maybe redis or mongodb, which may happen. The docs say that if we need to use a custom driver then it needs to implement the SessionHandlerInterface. So we can implement any driver and just implement the interface, and specify which one we want to use. The DB store is good for now.

**Comment by AlexeiaW** [comment]
The service makes sense at the end of the day. Maybe it's just the terminology that's going to cause some confusion. We can access the session in the resolvers, but also using the service.

Thus keeping SessionService

**Comment by luke-siedle** [comment]
@AlexeiaW I think leave this to me to refactor the session service code since I wrote it :)
